src/components/video-card/VideoCard.jsx

```diff
@@ -0,0 +1,46 @@
+import "./video-card.css";
+import { MdOutlineWatchLater, MdPlaylistAdd } from "react-icons/md";
+import { AiOutlineLike } from "react-icons/ai";
+
+const VideoCard = ({ videoData }) => {
+   return (
+      <div className="video-card">
+         <img
+            src={`http://img.youtube.com/vi/${videoData._id}/maxresdefault.jpg`}
+            alt="video thumbnail"
+            className="res-img"
+         />
+         <div className="card-footer">
+            <img
+               className="avatar-circle avatar-sm"
+               src={videoData.channelThumbnail}
+               alt="channel thumbnail"
+            ></img>
+            <div className="icon-wrapper">
+               <span>
+                  <AiOutlineLike size="24px" />
+               </span>
+               <span>
+                  <MdOutlineWatchLater size="24px" />
+               </span>
+               <span>
+                  <MdPlaylistAdd size="24px" />
+               </span>
```

Comment:
You can make the sizes generic as youre repeating

Reply:
Created a variable for them. Also, I have a default size but here in card need small icons, so used size prop here to override the default one.